drivers/sensor/ina260/ina260.c

```diff
@@ -0,0 +1,159 @@
+#include "ina260.h"
+#include <zephyr/device.h>
+#include <zephyr/devicetree.h>
+#include <zephyr/drivers/i2c.h>
+#include <zephyr/drivers/sensor.h>
+#include <zephyr/logging/log.h>
+#include <zephyr/sys/byteorder.h>
+
+#define DT_DRV_COMPAT ti_ina260
+
+LOG_MODULE_REGISTER(INA260, CONFIG_SENSOR_LOG_LEVEL);
+
+static int ina260_reg_read(const struct device *dev, uint8_t reg_addr,
+                           uint16_t *reg_data) {
+  const struct ina260_config *cfg = dev->config;
+  uint8_t rx_buf[2];
```

Comment:
```suggestion
  uint8_t rx_buf[2] = {0};
```

Good practice to 0 out your buffers in general